lib/utils/args.js

```diff
@@ -54,21 +54,24 @@ function _normaliseFlagsAndArs(argsIntoCommand, cmd) {
     // bail on old style commands
     return argsIntoCommand;
   }
-  var processArgs = _.clone(process.argv);
+
+  const normalisedArgs = _.clone(argsIntoCommand);
+  
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove the useless trailing whitespaces at the end of this line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ATrailingWhitespace)